cli/src/ios/update.ts

```diff
@@ -56,13 +51,62 @@ async function updateIOSCocoaPods(config: Config, plugins: Plugin[], deployment:
   }
   await handleCordovaPluginsJS(cordovaPlugins, config, platform);
   await checkPluginDependencies(plugins, platform);
-  await generateCordovaPodspecs(cordovaPlugins, config);
-  await installCocoaPodsPlugins(config, plugins, deployment);
+  if ((await checkPackageManager(config)) === 'SPM') {
+    await generateCordovaPackageFiles(cordovaPlugins, config);
+    await generatePackageFile(config, plugins);
+  } else {
+    await generateCordovaPodspecs(cordovaPlugins, config);
+    await installCocoaPodsPlugins(config, plugins, deployment);
+  }
   await logCordovaManualSteps(cordovaPlugins, config, platform);
 
   const incompatibleCordovaPlugins = plugins.filter((p) => getPluginType(p, platform) === PluginType.Incompatible);
   printPlugins(incompatibleCordovaPlugins, platform, 'incompatible');
-  await checkPlatformVersions(config, platform);
+}
+
+async function generateCordovaPackageFiles(cordovaPlugins: Plugin[], config: Config) {
+  cordovaPlugins.map((plugin: any) => {
+    generateCordovaPackageFile(plugin, config);
+  });
+}
+
+async function generateCordovaPackageFile(p: Plugin, config: Config) {
+  const iosPlatformVersion = await getCapacitorPackageVersion(config, config.ios.name);
+  const iosVersion = getMajoriOSVersion(config);
+  const headerFiles = getPlatformElement(p, platform, 'header-file');
+  let headersText = '';
+  if (headerFiles.length > 0) {
+    headersText = `,
+            publicHeadersPath: "."`;
+  }
+
+  const content = `// swift-tools-version: 5.9
+
+import PackageDescription
+
+let package = Package(
+    name: "${p.name}",
```

Comment:
If the Cordova plugin is mixed source, this will fail, right?

Reply:
That's why the PR is called "Initial Cordova plugin SPM support", not all plugins will work and future improvements will need to be done, it worked with the ones I tested, but I couldn't find any with mixed sources as it's not as common as in Capacitor plugins where all of them had both Objective-C and Swift code, on Cordova most of them are Objective-C and a few of them use just Swift and mixed source is uncommon.